packages/pretty-format/src/plugins/ImmutablePlugins.js

```diff
@@ -4,23 +4,17 @@
  * This source code is licensed under the BSD-style license found in the
  * LICENSE file in the root directory of this source tree. An additional grant
  * of patent rights can be found in the PATENTS file in the same directory.
+ *
  * @flow
  */
- 
-'use strict';
 
-const ImmutableOrderedSetPlugin = require('./ImmutableOrderedSet');
-const ImmutableListPlugin = require('./ImmutableList');
-const ImmutableMapPlugin = require('./ImmutableMap');
-const ImmutableOrderedMapPlugin = require('./ImmutableOrderedMap');
-const ImmutableSetPlugin = require('./ImmutableSet');
-const ImmutableStackPlugin = require('./ImmutableStack');
+'use strict';
 
 module.exports = [
-  ImmutableOrderedSetPlugin,
-  ImmutableListPlugin,
-  ImmutableOrderedMapPlugin,
-  ImmutableMapPlugin,
-  ImmutableSetPlugin,
-  ImmutableStackPlugin,
+  require('./ImmutableOrderedSet'),
+  require('./ImmutableList'),
+  require('./ImmutableMap'),
+  require('./ImmutableOrderedMap'),
```

Comment:
Good point @thymikee, the order matters here.
I would suggest making sure `ImmutableMap` doesn't have `@@__IMMUTABLE_ORDERED__@@` .
Same thing for `ImmutableOrderedSet` and `ImmutableSet`.
Can I work on this PR?